File: challenge-180/js/ch-1.js
#!/usr/bin/env node

'use strict';

function find_index(string) {
	let u_index;

	const uniq = string.split('').find(
		(chr, index) => {
			u_index = string.indexOf(chr);
			return u_index === string.lastIndexOf(chr) ? true : false;
		}
	);

	return uniq ? [ uniq, u_index ] : [];
}

[
	"Perl Perl Perl",
	"Perl Weekly Challenge",
	"Long Live Perl",
	"This is Hyper Perly",
	"I Love Perl",
].forEach( (i) => console.log(find_index(i)) );

File: challenge-182/perl/ch-1.pl
#!/usr/bin/env perl

use strict;
use warnings;

use Test::More;

while ( my $line = <DATA> ) {
	my @array = split m/, /, $line
}

my $expected = [2, 4];

is_deeply( $got, $expected, "");

done_testing(1);

__DATA__
5, 2, 9, 1, 7, 6
4, 2, 3, 1, 5, 0
